test/functional-tests/features/fieldTypes.feature

```diff
@@ -40,8 +41,97 @@ Feature: Set of scenarios to check we can read and write to all field data types
      #| Fixed-List|
       | Phone UK  |
       | Yes-No    |
-#      | Collection| TODO
-#      | Complex   | TODO
+      | Collection|
+      | Complex   |
+
+
+    @validation
+  Scenario: Validation: cannot progress to next page without filling in mandatory field
+    Given a case with a Mandatory field exists
+    When I do not fill in the Mandatory field
+    Then the 'Continue' button will be disabled
+
+    @validation
+  Scenario: Validation: string cannot be entered into a number field
+    Given a case type containing every field type exists
+    When I enter 'iamastring' into the 'number' field
+    Then no text will appear in the number field
+
+    @validation
+  Scenario: Validation: invalid phone number is not accepted
+    Given a case type containing every field type exists
+    And I enter '11111111111' into the 'Phone UK' field
+    When I click the Continue button
+    Then there will be validation errors
+    And the 'Continue' button will be disabled
+
+    @validation
+  Scenario: Validation: invalid email is not accepted
+    And a case type containing every field type exists
+    And I enter 'invalidemail.com' into the 'email' field
+    When I click the Continue button
+    Then there will be validation errors
+    And the 'Continue' button will be disabled
+
+    @validation
+  Scenario: Validation: invalid Money amount is not accepted
+    And a case type containing every field type exists
+    And I enter '-1' into the 'Money-gbp' field
+    Then there will be a field validation error
+    And the 'Continue' button will be disabled
+
+    @validation
+  Scenario: Validation: invalid date is not accepted
+    Given a case type containing every field type exists
+    And I enter '01201990' into the 'Date' field
+    When I click the Continue button
+    Then there will be validation errors
+    And the 'Continue' button will be disabled
+
+    @validation @regex
+  Scenario: Validation: breaking REGEX validation stops progression of a case
+    Given a case type containing a regex validated field exists
+    And I enter 'lowercaseisinvalid' into the 'text' field
+    When I click the Continue button
+    Then there will be validation errors
+    And the 'Continue' button will be disabled
+
+    @validation @bug
+  Scenario: Validation: can continue after correcting email validation failure
```

Comment:
guess you meant Phone UK here and Email in the next scenario

Reply:
ahh yes good spot